web/src/main/js/dynamic/DistributionConfiguration.js

```diff
@@ -11,6 +11,7 @@ import {
 import ConfigButtons from 'component/common/ConfigButtons';
 import { Modal } from 'react-bootstrap';
 import JobCustomMessageModal from 'dynamic/JobCustomMessageModal';
+import StatusMessage from '../field/StatusMessage';
```

Comment:
Please update this import to an absolute path.

Reply:
Is there a way to make Intellij do this automatically?